example/android/kotlinlib/3-compose-screenshot-tests/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is the app sources in `example/android/kotlinlib/3-compose-screenshot-tests/app/` based on any existing template or project generator? If so we should leave a comment mentioning that somewhere

Reply:
thanks! I've just pushed the sample AGP project in github and linked it here